<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0"><channel><title>MongoDB - 标签 - Spaceack's Blog</title><link>https://spaceack.github.io/tags/mongodb/</link><description>MongoDB - 标签 - Spaceack's Blog</description><generator>Hugo -- gohugo.io</generator><language>zh-CN</language><managingEditor>spaceack@qq.com (Spaceack)</managingEditor><webMaster>spaceack@qq.com (Spaceack)</webMaster><copyright>This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</copyright><lastBuildDate>Thu, 08 Nov 2018 21:00:00 +0000</lastBuildDate><atom:link href="https://spaceack.github.io/tags/mongodb/" rel="self" type="application/rss+xml"/><item><title>MongoDB常用操作命令</title><link>https://spaceack.github.io/20181107-mongodb%E5%B8%B8%E7%94%A8%E5%91%BD%E4%BB%A4/</link><pubDate>Thu, 08 Nov 2018 21:00:00 +0000</pubDate><author>Spaceack</author><guid>https://spaceack.github.io/20181107-mongodb%E5%B8%B8%E7%94%A8%E5%91%BD%E4%BB%A4/</guid><description><![CDATA[1 2 3 4 5 6 7 import pymongo myclient = pymongo.MongoClient(host=&#34;localhost&#34;, port=27017) mydb = myclient.database_name collection_obj = mydb.collection_name # upsert 是否插入新数据，如果不存在则插入，存在则更新 collection_obj.update_one({&#34;id&#34;: &#34;1&#34;}, {&#34;$set&#34;: {&#34;id&#34;: &#34;1&#34;, &#34;age&#34;: 18}}, upsert=True) 统计 collection信息 db.getCollection('collection_name').stats()]]></description></item><item><title>RCA-MongoDB数据写入失败</title><link>https://spaceack.github.io/20181107-rca-mongodb%E6%95%B0%E6%8D%AE%E5%86%99%E5%85%A5%E5%A4%B1%E8%B4%A5/</link><pubDate>Wed, 07 Nov 2018 21:00:00 +0000</pubDate><author>Spaceack</author><guid>https://spaceack.github.io/20181107-rca-mongodb%E6%95%B0%E6%8D%AE%E5%86%99%E5%85%A5%E5%A4%B1%E8%B4%A5/</guid><description>问题现象 程序崩溃，提示MongoDB写入失败，无法再连起。 分析原因 1.首先想到分析mongoDB日志记录 通过 cat /etc/mongod.conf 找到日志所在目录 /var/log/mongodb/mongod.log 1 2 3 4 2018-11-07T16:50:44.165+0800</description></item></channel></rss>